src/main/kotlin/org/wycliffeassociates/otter/common/domain/resourcecontainer/project/usfm/UsfmProjectReader.kt

```diff
@@ -33,29 +46,52 @@ class UsfmProjectReader: IProjectReader {
         return Pair(result, projectTree)
     }
 
+    private fun constructTreeFromZip(container: ZipResourceContainer, project: Project): Pair<ImportResult, Tree> {
+        // Find the appropriate zip entry and use it to construct the project tree
+        container.zip.entries().toList().filter {
+            project.path.contains(it.name)
```

Comment:
Could this cause trouble if one project name is a substring of another?

Reply:
I don't know how to handle this properly right now, since we don't have a .resolve method for zipfiles like we do for files. Every project.path does appear to start with "./" so one ugly way to avoid the substring issue would just be to add "./" to the start of the zip entry name and do an == compare with project.path.